frappe/utils/response.py

```diff
@@ -71,7 +71,7 @@ def as_txt():
 def as_raw():
 	response = Response()
 	response.mimetype = frappe.response.get("content_type") or mimetypes.guess_type(frappe.response['filename'])[0] or "application/unknown"
-	response.headers["Content-Disposition"] = ("attachment; filename=\"%s\"" % frappe.response['filename'].replace(' ', '_')).encode("utf-8")
+	response.headers["Content-Disposition"] = (f"{frappe.response.get('display_content_as') or 'attachment'}; filename=\"%s\"" % frappe.response['filename'].replace(' ', '_')).encode("utf-8")
```

Comment:
Seems like we're using two different string formatting methods in a single line. Replacing the `%s` string usage would be nice.

Reply:
Sure, will replace it to f-string format.

Reply:
@gavindsouza Can you please verify?

Reply:
I think double quotes are missing. Does it work this way?

Also, can you add steps to use this and documentation for this on [frappe/frappe_docs](https://github.com/frappe/frappe_docs)?

Reply:
Missed the quotes in the filename, added.

Documentation, trying to add.

Steps to use:
Last statements of the API should be, assuming API is generating Image,PDF
	frappe.local.response.filecontent = filecontent
	frappe.local.response.filename = filename
	frappe.local.response.type = "download"
	frappe.local.response.display_content_as = 'inline' or 'attachment'

inline will show in browser itself
attachment will download it